pkg/test/runner/runner.go

```diff
@@ -113,15 +113,11 @@ func (r *Runner) runFnEval() error {
 	}
 	if r.testCase.Config.EvalConfig.Image != "" {
 		kptArgs = append(kptArgs, "--image", r.testCase.Config.EvalConfig.Image)
-	} else if r.testCase.Config.EvalConfig.ExecPath != "" {
-		kptArgs = append(kptArgs, "--exec-path", r.testCase.Config.EvalConfig.ExecPath)
+	} else if r.testCase.Config.EvalConfig.execUniquePath.Empty() {
```

Comment:
You probably want not-empty here ? `!...Empty()`

Reply:
Yep, you are right. :)